config-connector/solutions/networking/helm/mci/README.md

```diff
@@ -0,0 +1,153 @@
+# Multi-cluster ingress
```

Comment:
We should call this "autoneg" or "standalone neg."

Multicluster ingress is already used to refer to the [native solution](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress-for-anthos).

Reply:
Changed